e2e/runner/sample-apps-shared/constants/sample-app-setup-configs.js

```diff
@@ -1,6 +1,8 @@
 const BRANCH_NAME = "main"; // Affects the `local` testing only. On CI is passed as an ENV variable.
 
 const BASE_ENV = {
+  WATCH:
+    process.env.SAMPLE_APP_ENVIRONMENT === "development" ? "true" : "false",
```

Comment:
I suppose `"true"` being a string is already correct as this is supposed to represent an environment variable

Reply:
yes, it's the value of the WATCH env variable